### DISCUSSION TITLE
Number of buildings in Italy

### DISCUSSION THREAD

**Discussion by tiffiw** [streetcomplete/StreetComplete#3987] [Feedback]
Is there any reason why the number of houses is deactivated in Italy? It would be great to be able to map them, too. And indeed, numbers of houses in Italy are existent...
Many thanks for this great app 👍

**Comment by Helium314** [comment]
See #714 and https://lists.openstreetmap.org/pipermail/talk-it/2018-July/063712.html for why it is disabled
